5.3.a_perfect_dish_to_share.py

```diff
@@ -0,0 +1,33 @@
+# מנה מושלמת לחלוקה
+import math
+
+
+def is_perfect_num(num):
+    _sqrt = math.sqrt(num)
+
+    sum_dividers = 0
+
+    for i in range(2, math.ceil(_sqrt)):
+        if num % i == 0:
+            sum_dividers = sum_dividers + i + num / i
+
+    if _sqrt == int(_sqrt):
+        sum_dividers = sum_dividers + _sqrt
+
+    return sum_dividers + 1 == num
+
+
+def perfect_number_generator():
+    num = 1
```

Comment:
Most times when writing in large code, num can mean a lot of things or it could have plenty of usage, so it is better to practice giving better variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing